lib/hydra/validations/presence.rb

```diff
@@ -0,0 +1,18 @@
+require 'hydra/validations/enumerable_behavior'
+module Hydra
+  module Validations
+    class PresenceValidator < ActiveModel::Validations::PresenceValidator
+      def validate_each(record, attribute, values)
+        record.errors.add(attribute, options[:message] || :invalid) if Array.wrap(values).blank?
```

Comment:
I believe this will work (though the tests should validate)

```ruby
values = Array.wrap(values)
record.errors.add(attribute, options[:message] || :invalid) unless values.none?(&:blank)
```